streaming/src/test/scala/org/apache/spark/streaming/TestSuiteBase.scala

```diff
@@ -104,6 +106,77 @@ class TestOutputStreamWithPartitions[T: ClassTag](parent: DStream[T],
 }
 
 /**
+ * This is an interface that can be used to block until certain events occur, such as
```

Comment:
I'll fix this up, along with my "two spaces after periods" style in the block comments.